### ISSUE TITLE
Not an issue but a question due to lack of docs.

### ISSUE THREAD

**Issue by mudapanda2** [microsoft/antares#364]
Hello there, is it possible to use this as a means to compile .h and .cpp HIP code on windows for creating GPU shader kernels? fatbins and cubins. I mean from pre-existing code to define the shaders.

The simplest way to explain it is Windows has no means to compile HIP c++ as far as I know, perhaps hiprtc workarounds, make some magic with oneAPI but so far this has looked the most promising. 

If I could feed it amdhip64.dll in WSL and ask it to compile .cpp with provided headers and return fatbin to compile a renderer then I would be awestruck

**Comment by ghostplant** [comment]
Yes, this is exactly how it works at present. But it is not designed to directly compile HIP code written in tensorflow/torch.
